### PR TITLE
New style for window resizer

### PR DESCRIPTION
**Pull request by felipeerias**
Improve the visual appearance of the window resizer by adding a space between the resizing bars and the window, hiding the handles, increasing the touch area, and using theme colors.

### CODE REVIEW

**Comment by felipeerias** [comment]
Screen recording and image:

https://github.com/user-attachments/assets/c5bf8820-c3c3-47b3-a1d1-c6a10eb4b568

Screenshot_com igalia wolvic dev_2024 11 27-20 11 36 712_633

One problem with the current implementation is that the hover/click areas are linked to the handles (invisible in this PR) and not the resize bars.

I think that we can fix that problem, although tbh it is frustratingly hard to do major modifications to `WidgetResizer.cpp`.

**Comment by svillar** [comment]
Why removing the resizing circles? I think they look good

**Comment by felipeerias** [comment]
@svillar  I have updated the visuals to show the round handlers again:

Screenshot_com igalia wolvic dev_2024 12 02-11 34 37 793_544

File: app/src/main/cpp/WidgetResizer.cpp [hunk 1]
```diff
@@ -33,18 +33,29 @@ struct ResizeBar;
...
+static const float kHandleRadius = 0.04f;
+// Radius of the hover/click area for the handles is kHandleRadius * kTouchRatio.
+static const float kTouchRatio = 6.0f;
 static const vrb::Vector kDefaultMinResize(1.5f, 1.5f, 0.0f);
 static const vrb::Vector kDefaultMaxResize(8.0f, 4.5f, 0.0f);
-static vrb::Color kDefaultColor(0x2BD5D5FF);
-static vrb::Color kHoverColor(0xf7ce4dff);
-static vrb::Color kActiveColor(0xf7ce4dff);
+// Colors for the handles.
+    static vrb::Color kDefaultHandleColor(0xE2E6EBFF);
```
Comment (svillar) [inline app/src/main/cpp/WidgetResizer.cpp:52]: Nit: extra blanks

**Review by svillar — APPROVED** [review]
Looks good, I fix the minor nit before landing

**Pull request merged**